corehq/apps/userreports/filters/__init__.py

```diff
@@ -70,3 +90,20 @@ def __init__(self, expression, operator, reference_expression):
 
     def __call__(self, item, context=None):
         return self.operator(self.expression(item, context), self.reference_expression(item, context))
+
+    def __str__(self):
+        return "{} {} '{}'".format(str(self.expression),
+                                   OPERATOR_DISPLAY[self.operator.func_name],
+                                   str(self.reference_expression))
+
+
+class NamedFilter(Filter):
+    def __init__(self, filter_name, filter):
+        self.filter_name = filter_name
+        self.filter = filter
+
+    def __call__(self, item, context=None):
+        return self.filter(item, context)
+
+    def __str__(self):
+        return "NamedF:{}".format(self.filter_name)
```

Comment:
`NamedF`?  You could probably just go with something like:
`'"{}" filter` or `'named filter: "{}"'`